src/pages/EditMonitor.vue

```diff
@@ -1243,6 +1254,16 @@ message HealthCheckResponse {
 
                         this.monitor = res.monitor;
 
+                        // Handling for when friendly name isn't set
+                        if (
+                            this.monitor.name === this.monitor.url
```

Comment:
```suggestion
                        if (this.monitor.name === this.monitor.url
```

Reply:
Personally I prefer it on a new line because it fits better with how we would write function arguments or arrays that split over new lines.